sycl/test/sub_group/generic-shuffle.cpp

```diff
@@ -0,0 +1,214 @@
+// UNSUPPORTED: cuda
+// CUDA compilation and runtime do not yet support sub-groups.
+//
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUNx: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
```

Comment:
It seems interesting to launch this test on host, but don't launch on GPU.

I expect it to exit at the very beginning for host device, as it doesn't support sub-groups - is there any reason to launch it then?
And why run on GPU is disabled?

Reply:
@vladimirlaz, can you help answer this one?

All the other sub-group tests are currently run on the host.  My guess is that this is so that if/when the host device claims to support sub-groups the tests will automatically start running.

`attributes.cpp` and `shuffle.cpp` use `RUNx` for the GPU, and because I based this test on `shuffle.cpp` I inherited it here.  I didn't realize that those tests or this new one would be disabled.  I'll update this test -- should we update the others as well?

Reply:
> My guess is that this is so that if/when the host device claims to support sub-groups the tests will automatically start running.

Ok, good point

>  I didn't realize that those tests or this new one would be disabled. I'll update this test -- should we update the others as well?

It would be good if you try that. I suppose that everything should be fine and I'm okay with enabling this test even in scope of this PR